src/molecules/lists/IconList/index.js

```diff
@@ -4,35 +4,53 @@ import classnames from 'classnames';
 
 import Icon from 'atoms/Icon';
 import Text from 'atoms/Text';
+import Tooltip from 'atoms/Tooltip';
 import styles from './icon_list.module.scss';
 
 function IconList(props) {
   const {
     className,
     data,
-    type,
     font,
+    type,
+    itemSpacing
   } = props;
 
   return (
     <ul className={classnames(styles['icon-list'], className)}>
       {
         data.map( (item, idx) =>
           <li
-            className={styles['item']}
+            className={styles.item}
             key={`iconlist-item-${idx}`}
+            style={{
+              marginBottom: itemSpacing
+            }}
           >
-            <Icon
-              className={styles['icon']}
-              icon={item.icon}
-            />
-
             <Text
-              className={styles['text']}
+              className={styles.text}
               type={type}
               font={font}
             >
+              <Icon
+                className={styles.icon}
+                icon={item.icon}
+                inline
+              />
               {item.text}
+              {
+                item.tooltip ?
+                  <Tooltip
+                    className={styles.tooltip}
+                    left
+                    inline
+                  >
+                    {item.tooltip}
+                  </Tooltip> : ''
```

Comment:
☝️  I had to do this with a ternary instead of the more idiomatic `bool && component` structure because if you have a child and a `null` inside of a `Text` component, the app breaks and gets very sad. :(